src/nvim/version.c

```diff
@@ -1106,7 +1112,8 @@ static void do_intro_line(int row, char_u *mesg, int attr)
         clen += byte2cells(p[l]);
       }
     }
-    screen_puts_len(p, l, row, col, *p == '<' ? hl_attr(HLF_8) : attr);
+    assert(row < INT_MAX && col < INT_MAX);
```

Comment:
`<=` better than `<`.
Yes, probably `screen_puts_len` would need refactoring. To keep this PR small and self-contained, it's ok to leave it that way for now.